README.rst

```diff
@@ -124,3 +124,33 @@ Use  ``--no-verify`` to disable verification for APROM or data flash. To disable
 ::
 
     $ evic-usb upload --no-verify aprom --no-verify dataflash firmware.bin
+
+Reset the device:
+
+::
+
+    $ evic-usb reset
+
+Dump any part of the flash memory:
```

Comment:
I would add that `fmc-read`, `time` and `screenshot` are specific to your FW. `reset` is good.